src/__tests__/createTheming.test.js

```diff
@@ -242,4 +242,34 @@ describe('createTheming', () => {
     );
     expect(render.mock.calls.length).toEqual(2);
   });
+
+  it('It doesnt mutate existing theme', () => {
+    const Checker1 = withTheme(({ theme }) => {
+      expect(theme).toEqual({
+        ...lightTheme,
+        primaryColor: 'red',
+      });
+      return null;
+    });
+
+    const Checker1WithTheme = withTheme(Checker1);
+
+    const Checker2 = withTheme(({ theme }) => {
+      expect(theme).toEqual({
+        ...lightTheme,
+        primaryColor: '#ffcaaa',
```

Comment:
I think instead of these, you can do something like `const originalTheme = { ...lightTheme }` at the top and compare to that so you don't have to type the primary color value again.